beacon-chain/core/state/transition_no_verify_sig.go

```diff
@@ -174,16 +174,11 @@ func ProcessBlockNoVerifyAnySig(
 	}
 
 	blk := signed.Block
-	body := blk.Body
```

Comment:
We can also remove `helpers.VerifyNilBeaconBlock()` block at line 170, as it is checked in `ProcessBlockForStateRoot()` and will be caught.